reference/tools/gnu/pkgconfigdeps.rst

```diff
@@ -139,6 +141,32 @@ renamed:
         pc.generate()
 
 
+build_context_folder
+^^^^^^^^^^^^^^^^^^^^
+
+*New since Conan 2.2.0*
+
+Same concept as the quoted ``build_context_suffix`` attribute above, but this is meant to specify a folder to save
+the `*.pc` files created by all those build requirements listed in the ``build_context_activated`` one:
```

Comment:
I think it would be better to not reference the deprecated attribute to document this one, just explaining this as if the other one did not exist as one day we will remove that one.

Reply:
Reordered